index.bs

```diff
@@ -4,9 +4,9 @@ Group: W3C TAG
 Shortname: design-principles
 Repository: w3ctag/design-principles
 Status: DREAM
-Editor: Travis Leithead, Microsoft, travil@microsoft.com
```

Comment:
Why is this change in here? Did something go wrong with the rebase?

Reply:
Yeah, moved to Former Editor: - already part of master

Reply:
If it's already in master, why is it showing up in this diff?

Reply:
Added the dropped 'a' from @annevk change and got rid of the editor's list change which is in master and not sure why it was flagged as diff here...